pages/[category]/[...subcategory].tsx

```diff
@@ -98,11 +99,18 @@ const SubCategory: NextPage<PageProps> = ({ subcategory }) => {
       <TopBar className="shadow-black-500/50 fixed top-[76px] z-30 flex w-full -translate-x-4 items-center bg-gray-100 px-4 pt-6 pb-4 shadow-xl dark:bg-gray-900 md:hidden" />
       <div
         data-custom="restrict-click-outside"
-        className="relative min-h-[calc(100%-68px)] w-full pt-[85px] pb-4 md:min-h-[calc(100%-76px)] md:px-10 md:pt-10"
+        className=" relative min-h-[calc(100%-68px)]  w-full pt-[85px] pb-4 md:min-h-[calc(100%-76px)] md:px-10 md:pt-10"
```

Comment:
Please, avoid adding extra spaces.

```suggestion
        className="relative min-h-[calc(100%-68px)] w-full pt-[85px] pb-4 md:min-h-[calc(100%-76px)] md:px-10 md:pt-10"
```